style.css

```diff
@@ -1,825 +1,810 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::after,
+*::before {
+	box-sizing: inherit;
+}
+
 body {
-  margin: 0;
-  font-family: Montserrat, sans-serif;
+	min-width: 320px;
+	overflow-x: hidden;
+	font-family: Montserrat;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)